.setup/dev-upgrade-h2.sh

```diff
@@ -119,7 +121,10 @@ update_apache() {
 
     info "Double check that HTTP/2 module is enabled"
     phpver=$(php -r 'print PHP_MAJOR_VERSION.".".PHP_MINOR_VERSION;')
+    #TODO: REMOVE THIS
+    set +e
     a2dismod "php${phpver}" mpm_prefork
+    set -e
```

Comment:
What is the rationale for disabling fail-on-error here?  If there's a good reason for it, that good reason should be documented in a detailed comment.  The same applies to each of the other exclusions as well.